subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/hived_algorithm.go

```diff
@@ -1105,8 +1123,12 @@ func getFewestOpporPhysicalCell(cl CellList, suggestedNodeSet common.Set) *Physi
 		}
 	}
 	if fewestOpporSuggested == nil {
+		if fewestOpporCell != nil {
+			klog.Infof("Returning a cell NOT within suggested nodes: %v", fewestOpporCell.nodes[0])
```

Comment:
>fewestOpporCell.nodes[0] [](start = 65, length = 24)

Why only log the first node? It may not be the node not in suggested.

   nodes, _ := pc.GetPhysicalPlacement()
   for _, n := range nodes {
    if !suggestedNodeSet.Contains(n) {
     allNodesInSuggested = false
     break
    }
   }